scraper/scraper.py

```diff
@@ -174,13 +176,15 @@ def centre_iterator():
     url = "https://www.data.gouv.fr/fr/datasets/r/5cb21a85-b0b0-4a65-a249-806a040ec372"
     response = requests.get(url)
     response.raise_for_status()
-
     reader = io.StringIO(response.content.decode('utf8'))
     csvreader = csv.DictReader(reader, delimiter=';')
     for row in csvreader:
-        yield row
+        if not 'doctolib' in row['rdv_site_web']:
+            yield row
     for centre in ordoclic_centre_iterator():
         yield centre
+    for centre in doctolib_centre_iterator():
```

Comment:
Peut-être stocker une liste d'itérateur et les balancer au lieu d'avoi trois for consécutifs?